### PR TITLE
Feature: use tailwindcss

### PR DESCRIPTION
**Pull request by ledsi**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)
- [ ] Documentation update
- [ ] Bug fix
- [ ] New feature
- [x] Other, please explain:

#### What changes did you make? (Give an overview)
Updated admin panel and pages that belong to admin features with tailwindcss and gave them a new fresh look.
Affected pages are:
- admin panel
- login
- diskusage
- debug
- manual

This update cleans up the styling, the generated php code for frontend rendering and a bit js rework and updates.
Also implements tailwindcss in gulp so that the use of css classes for styling is not longer needed.

#### Is there anything you'd like reviewers to focus on?
@andi34 this update took a while and removed a few "custom stylings and settings" for the admin area but i think its not necessary for a photobooth admin panel. I did not remove the custom settings at all, maybe you can have a look at this. the focus is now a more clean ui and an easy and fast styleable UI without writing css.
Because of there were so many files changed please have a look at all the changes / documentations and functions in admin panel. Hopefully there is nothing screwed up and all works well. 

Maybe we should add more reviewers before merging this changes -> if you like the updated and we want to implement it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ _Latest development version:_
 - Gallery based on [PhotoSwipe v5](https://github.com/PhotoboothProject/PhotoSwipe)
 - Standalone Gallery based on PhotoSwipe v5 ([localhost/gallery.php](http://localhost/gallery.php)).
 - Standalone Slideshow based on PhotoSwipe v5 ([localhost/slideshow](http://localhost/slideshow)).
-- Access login via [localhost/login](http://localhost/login) instead [localhost/login.php](http://localhost/login.php).
+- Access login via [localhost/login](http://localhost/login) instead [localhost/login](http://localhost/login).
```
Comment (andi34) [inline README.md]: Remove the `instead [localhost/login.php](http://localhost/login.php).` completely (was old Location some years ago).

**Comment by andi34** [comment]
Wow, Great! Yeah, the adminpanel was one of the things I'd liked to look at in future too, haha. Great you took the advantage! I hope to get the api and classes rework finished next week and find time to look at your PR closer!
The remotebuzzer changes are the only things I need/like.to merge before the big re-write

**Comment by andi34** [comment]
Fixed getting content inside debugpanel.

Looks really nice in general 😊
Can we make the colors adjustable?

On mobile view the top elements get lost if i select a category from the list
Screenshot_20230521-133015

**Comment by andi34** [comment]
Anything from the src/sass can be removed?

**Comment by ledsi** [comment]
> Anything from the src/sass can be removed?

old sass files removed


> Fixed getting content inside debugpanel.
> 
> Looks really nice in general 😊 Can we make the colors adjustable?
> 
> On mobile view the top elements get lost if i select a category from the list Screenshot_20230521-133015

admin brand color is now adjustable via admin panel

mobile header anchor issue is fixed

**Comment by andi34** [comment]
Great! Will test asap!
Is selectize still used or can it be removed from the dependencies in package.json?

File: src/js/admin.js [hunk 2]
```diff
@@ -1,355 +0,0 @@
-/* globals photoboothTools */
```
Comment (andi34) [inline src/js/admin.js:1]: Remove completely? Where are those functions handled now?

File: src/js/adminshortcut.js [hunk 3]
```diff
@@ -1,17 +0,0 @@
-/* globals photoboothTools */
```
Comment (andi34) [inline src/js/adminshortcut.js]: Same here?

Comment (ledsi) [reply src/js/adminshortcut.js]: src/js/admin/*

File: src/js/admin.js (hunk 2 above)
Comment (ledsi) [reply src/js/admin.js:1]: admin js got its own folder "src/js/admin" which handles all the admin js stuff now. files are split and more function based and will be concat on gulp process to "resources/js/main.admin.js"

File: src/js/adminshortcut.js (hunk 3 above)
Comment (andi34) [reply src/js/adminshortcut.js]: https://github.com/PhotoboothProject/photobooth/blob/ce14c3e2863f73b2df309d915018e70114cad024/index.php#L240

Guess also on livechroma.php

Comment (ledsi) [reply src/js/adminshortcut.js]: what do you mean? i didn't touch livechroma.php at all

Comment (andi34) [reply src/js/adminshortcut.js]: The src/js/adminshortcut.js is removed but index for example uses it.

Comment (ledsi) [reply src/js/adminshortcut.js]: oh yeah, your right. thanks, I missed that those pages are using it too.

**Comment by andi34** [comment]
Going to rebase in next hours.

**Pull request merged**